### DISCUSSION TITLE
ERROR Exit code was 1603!

### DISCUSSION THREAD

**Discussion by Valinor** [ScoopInstaller/Main#2801] [General]
When you encounter error 1603, check theses informations before reporting : 

1. You can try running
```
scoop cache rm *
scoop install myproblematickpackage
```

2. You can try running
```
scoop config MSIEXTRACT_USE_LESSMSI $true
scoop uninstall myproblematickpackage
scoop install myproblematickpackage
```

1603 error : https://docs.microsoft.com/en-us/troubleshoot/windows-server/application-management/msi-installation-error-1603

1603 error information : https://www.get-itsolutions.com/how-to-fix-error-1603-fatal-error-during-installation

**Comment by haussmann** [comment]
It also happens for example if an MSI is executed by the installer that requests admin rights via UAC and the user denies it.

Workaround in the nonportable bucket is to check for admin status in the installer / uninstaller script.
